fastembed/rerank/cross_encoder/onnx_text_model.py

```diff
@@ -0,0 +1,28 @@
+from typing import Sequence
+from fastembed.common.onnx_model import OnnxModel, OnnxProvider
+from optimum.onnxruntime import ORTModelForSequenceClassification
+from transformers import AutoTokenizer
+
+class OnnxCrossEncoderModel(OnnxModel):
```

Comment:
`OnnxModel` children should implement its methods, like `onnx_embed`, and avoid introducing new methods for the same functionality (encode) if possible